draft-ietf-dnsop-ns-revalidation.mkd

```diff
@@ -318,12 +320,12 @@ Resolvers following referrals from a rogue name server, that do not revalidate t
 The higher up the DNS tree, the more impact such an attack has.
 An attacker controlling a rogue name server for the root has potentially complete control over the entire domain name space and can alter all unsigned parts undetected.
 
-Revalidating referral and authoritative NS RRset responses enables to defend against the above described attack with DNSSEC signed infrastructure RRsets.
+Strictly revalidating referral and authoritative NS RRset responses (see {{strict}}), enables to defend against the above described attack with DNSSEC signed infrastructure RRsets.
```

Comment:
"enables to defend" doesn't sound grammatically correct to me.

My alternative wording: "enables the resolver to defend itself against ...."

Reply:
Resolved in 24be842